blogs/rtt/round_trip_time.md

```diff
@@ -0,0 +1,109 @@
+# Network Observability TCP Handshake Round Trip Time Developer Preview
+
+![logo](./images/rtt_logo.png)
+
+By: Dushyant Behl and Julien Pinsonneau
+
+In OCP ensuring efficient packet delivery is crucial for maintaining smooth
```

Comment:
Add comma.
`In OCP,`

Reply:
Done https://github.com/netobserv/documents/pull/54/commits/2a3dde7605b7231dc87de1f71eb93cb2d7b66266